### ISSUE TITLE
README assumes know about http-server

### ISSUE THREAD

**Issue by cgatian** [CanopyTax/single-spa-angular#36]
The instructions include running an http-server instance:

`npx http-server . -o`

Which shouldnt be possible unless you `npm i http-server`

**Comment by joeldenning** [comment]
`npx` comes installed with node and npm. It will download and install a library and then execute it as a cli.

The command in the documentation works without any extra configuration.

See https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b for more details

**Comment by cgatian** [comment]
Http-server isn't a dependency of the package though. My issue isn't npx it's you're referring to a package that doesn't exist

**Comment by joeldenning** [comment]
You didn't read the article I linked you to 😄. Additionally the readme no longer refers to npx or http-server, now that `ng serve` is working.

To give you a summary of what the article says, `npx http-server` will **install** http-server and then run it. Here's the http-server package on npm: https://www.npmjs.com/package/http-server. And here's the article I linked to earlier: https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b
